ui/imodel-components-react/src/imodel-components-react/timeline/SolarTimeline.tsx

```diff
@@ -544,18 +543,18 @@ export class SolarTimeline extends React.PureComponent<
             <Label htmlFor="speed">Timeline speed</Label>
           </VisuallyHidden>
           <Tooltip content={this._speedLabel}>
-            <select
-              className="solar-timeline_speed"
+            <Select
+              native
+              styleType="borderless"
               name="speed"
-              onChange={(e) => this._onSpeedChange(Number(e.target.value))}
+              onChange={(newValue) => this._onSpeedChange(Number(newValue))}
```

Comment:
where is `newValue` coming from? Would it be possible to convert it to a Number prior to this?

Reply:
I think iTwinUI `Select` already returns `newValue` as type `number` based on `options` values.  So converting it again is redundant and `Number()` can be removed.

Reply:
In case of a non-native Select, the value type is generic and could be a number w/o a need for the conversion. However, when `native` variant is used, the value is always typed as `string`.

Reply:
Fixed.